frontend/src/views/c-authorship.vue

```diff
@@ -1,13 +1,14 @@
 <template lang="pug">
 #authorship
-  .panel-title
-    span Code Panel
-  .toolbar--multiline
-    a(
-      v-if="activeFilesCount < selectedFiles.length",
-      v-on:click="toggleAllFileActiveProperty(true)"
-    ) show all file details
-    a(v-if="activeFilesCount > 0", v-on:click="toggleAllFileActiveProperty(false)") hide all file details
+  h2
```

Comment:
Was this `h2` added to provide padding or for semantic reasons? Because the `.toolbar--multiline` wouldn't be a heading right?

Reply:
Without the `h2`, the `Code Panel` will appear at the top of the screen, above the title.

![without h2](https://github.com/reposense/RepoSense/assets/110604064/012cc2fa-3729-4cd5-9093-a132a8c0a642)

If `.toolbar--multiline` is not included in `h2`, it appears on the line below instead.

![not in h2](https://github.com/reposense/RepoSense/assets/110604064/2f0ed407-0a6b-4353-bcea-f7ba67f1115b)

The expected behaviour is as follows:

![expected](https://github.com/reposense/RepoSense/assets/110604064/c8677e55-9ae2-4e5a-84d1-ef39518810a1)

Reply:
It may be possible to align by adding top padding for these elements instead of making them both h2, but this works well enough.